src/denyList.ts

```diff
@@ -0,0 +1,102 @@
+import { Redis } from "./types"
+import { RatelimitResponse } from "./types"
+import { Cache } from "./cache";
+
+
+const denyListCache = new Cache(new Map());
+
+/**
+ * Checks items in members list and returns true if any
+ * of them are in denyListCache.
+ * 
+ * @param members list of values to check against the cache
+ * @returns a member from the cache. If there is none, returns undefined
+ */
+export const checkDenyListCache = (members: string[]): string | undefined => {
+  return members.filter(
```

Comment:
`some()` would be a better choice here. If our goal is to 'get any of them'

Reply:
did it using find()

Reply:
Ah, sorry I thought you were just checking if its exists or not. Good call.